amp-pwa-reader/src/js/Article.js

```diff
@@ -262,6 +321,10 @@ class Article {
     document.scrollingElement.scrollTop = this._mainScrollY;
   }
 
+  urlProxy(url) {
+    return 'https://seed-octagon.glitch.me/' + encodeURIComponent(url);
```

Comment:
wait this is gone, no?

Reply:
So you would think!  But I did this and the server-side code completely separately.

I'll have to merge sooner or later.

Reply:
looking forward to it.